third_party/parser/cc/capi.cc

```diff
@@ -1,101 +1,69 @@
-#include <ruby_parser/capi.hh>
```

Comment:
i found a couple of files that were not having clang-format run on them

i landed those changes as a separate commit, if you want to ignore this noise

i haven't looked into why they're not getting formatted automatically (but it's probably not that important)